### PR TITLE
Fix F query filtration. Closes #174.

### LINKED ISSUE #174: Filtration with F objects fails on Django < 1.8
Simple queries are ok:

```python
>>> ModelWithVanillaMoneyField.objects.filter(money=F('money'))
[]
```

In more complex case it fails:

```python
>>> ModelWithVanillaMoneyField.objects.filter(second_money=F('money') * 2)
>                   to_append['_'.join([clean_name, 'currency'])] = F(get_currency_field_name(value.name))
E                   AttributeError: 'ExpressionNode' object has no attribute 'name'

djmoney/models/managers.py:116: AttributeError
```

I'll provide fix in next few days.

Similar issue - #101

### PR DESCRIPTION
**Pull request by Stranger6667**

### CODE REVIEW

**Comment by Stranger6667** [comment]
Unfortunately [https://travis-ci.org/django-money/django-money/jobs/108542712](this job) failed with some Travis-related error

**Comment by benjaoming** [comment]
rerunning the failed one

File: djmoney/utils.py [hunk 1]
```diff
@@ -1,3 +1,30 @@
...
+
+from moneyed import Money
+
+from ._compat import BaseExpression
+
 
 get_currency_field_name = lambda name: "%s_currency" % name
+
+
+def split_expression(expr):
```
Comment (benjaoming) [inline djmoney/utils.py]: shouldn't this live in compat? :)

Comment (Stranger6667) [inline djmoney/utils.py]: You're right :)

**Comment by benjaoming** [comment]
Great improvements! Thanks for another great PR, this is how it should be done!!

**Comment by Stranger6667** [comment]
Thank you, @benjaoming, I'm pleased to hear that :)

**Pull request merged**